test/ContextTests.ts

```diff
@@ -0,0 +1,35 @@
+import { getNormalizedBindingData, toRpcHttp } from '../src/Converters';
+import { expect } from 'chai';
+import * as sinon from 'sinon';
+import { FunctionRpc as rpc } from '../azure-functions-language-worker-protobuf/src/rpc';
+import 'mocha';
+
+describe('Context', () => {
+  it('normalizes binding trigger metadata', () => {
+    var mockRequest: rpc.TypedData$Properties = toRpcHttp({ url: "https://mock"});
+    var triggerDataMock: { [k: string]: rpc.TypedData$Properties } = {
+        "Headers": {
+            json: JSON.stringify({Connection: 'Keep-Alive'})
+        },
+        "Req": mockRequest,
+        "Sys": {
+            json: JSON.stringify({MethodName: 'test-js', UtcNow: '2018', RandGuid: '3212'})
```

Comment:
for completeness, add tests for other casing camelCased : 'testCamelCase', lowercased: 'testlowercase' etc